HadesAPI/dashboard/handlers.go

```
@@ -0,0 +1,234 @@
+	req, err := http.NewRequestWithContext(c.Request.Context(), http.MethodGet, target, nil)
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": "failed to build log request"})
+		return
+	}
+	resp, err := s.http.Do(req)
+	if err != nil {
+		slog.Warn("Log manager unreachable", "job_id", parsedID, "error", err)
+		c.JSON(http.StatusServiceUnavailable, gin.H{"error": "log service unavailable"})
+		return
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)